### PR TITLE
Add new meetups data.

### PR DESCRIPTION
**Pull request by josisusan**
Meetups data from:

16th June 2019
29th Oct 2019

### CODE REVIEW

File: js/meetup/data.js [hunk 1]
```diff
@@ -1,4 +1,76 @@
...
+      },
+      {
+        id: 2,
+        title: 'Jekyll Themes and zero-cost Hosting',
+        description: null,
+        speaker: 'Rajan Bhattarai',
+        twitter_username: 'cdrrazan',
+        github_username: 'cdrrazan',
+        designation: 'Full Stack Ruby on Rails Developer',
+        slides_link: '',
```
Comment (zoras) [inline js/meetup/data.js]: slides here https://meetup.cdrrazan.com/

File: js/meetup/data.js [hunk 2]
```diff
@@ -1,4 +1,76 @@
...
+    },
+    sessions: [
+      {
+        id: 1,
+        title: 'My Journey',
+        description: null,
+        speaker: 'Aram Bhusal',
+        twitter_username: 'phoenixwizard',
+        github_username: 'phoenixwizard',
+        designation: 'Principal Engineer, Zoomcar Full Stack',
```
Comment (zoras) [inline js/meetup/data.js]: `Prinicipal Engineer, Zoomcar`

File: js/meetup/data.js [hunk 3]
```diff
@@ -1,4 +1,76 @@
...
+    sessions: [
+      {
+        id: 1,
+        title: 'Build your free personal Developer website with Jekyll',
+        description: null,
+        speaker: 'Budha Ram Gurung',
+        twitter_username: 'coolbrg',
+        github_username: 'coolbrg',
+        designation: 'CTO, Siris Youth Foundation',
+        slides_link: '',
```
Comment (zoras) [inline js/meetup/data.js]: https://docs.google.com/presentation/d/1khnzL-VyH0CN_GJaZVSYrq-Dve5Tsbzv9Tsphu41adU/edit?usp=drivesdk

Comment (josisusan) [reply js/meetup/data.js]: The link ask for permission from the author. This is not a public url.

Comment (josisusan) [reply js/meetup/data.js]: @coolbrg could you make the link public?

**Pull request merged**